source/WebConfig/webconfig_internal.c

```diff
@@ -573,8 +574,9 @@ int requestWebConfigData(char **configData, int r_count, int index, int status,
 		}
 
 		// set callback for writing received data
+		dataVal = &data;
 		curl_easy_setopt(curl, CURLOPT_WRITEFUNCTION, write_callback_fn);
-		curl_easy_setopt(curl, CURLOPT_WRITEDATA, &data);
+		curl_easy_setopt(curl, CURLOPT_WRITEDATA, dataVal);
```

Comment:
void * typecast also should work, variable is not needed

Reply:
(void*)&data doesn't help, coverity still shows **BAD_SIZEOF** issue.
_Event bad_sizeof: Taking the size of "&data", which is the address of an object, is suspicious._
Hence retained same above approach.